bootstrap.sh

```diff
@@ -6,8 +6,10 @@ set -ev
 
 LATEST_PUPPET_PACKAGE=puppetlabs-release-pc1-trusty.deb
 
-MES_AIDES_BOOTSTRAP_FOLDER=/opt/mes-aides-bootstrap
-MANIFESTS_DESTINATION_FOLDER=$MES_AIDES_BOOTSTRAP_FOLDER/manifests
+PROVISIONING_FOLDER=/opt/mes-aides
+BOOTSTRAP_FOLDER=$PROVISIONING_FOLDER/bootstrap
+BOOTSTRAP_MANIFESTS_DESTINATION_FOLDER=$BOOTSTRAP_FOLDER/manifests
```

Comment:
> That's a lot of variables. I don't think it would be less readable to use `$BOOTSTRAP_FOLDER/$ MANIFESTS_SOURCE_FOLDER` dynamically in the few places where this variable is used, but I think it would decrease the cognitive load.